Fw/Time/test/ut/TimeTest.cpp

```diff
@@ -89,6 +89,13 @@ TEST(TimeTestNominal,CopyTest) {
 
 }
 
+TEST(TimeTestNominal,ZeroTimeEquality) {
+    Fw::Time time(TB_DONT_CARE,1,2);
```

Comment:
This code looks odd. We are using TB_DONT_CARE, but then the test says we do care. Maybe we can use TB_PROC_TIME instead? It's already used in a different test.